### DISCUSSION TITLE
Math mode in a table caption

### DISCUSSION THREAD

**Discussion by Andrew-S-Rosen** [typst/typst#4296] [Q&A] [answered]
The `caption` field of a `table` can be used to specify the caption. I would like to make the caption something like "My table description. The value of $m$ is 1." However, I can't figure out how to inject math mode into the string (so that I can use $m$ instead of m). Any suggestions on how to do something like this? Doing `caption: "My table description. The value of $m$ is 1." renders with the actual $ signs.

This is somewhat of a more general question about how to enter math mode from within a string.

**Comment by LaurenzV** [comment, marked as answer]
You mean the `caption` field of `figure`, right? If so, you should be able to just pass your caption as content instead of a string, so you can just write `[My table description. The value of $m$ is 1.]`

**Reply by Andrew-S-Rosen** [reply]
Ah, thank you very much! Yes, I did mean a `figure` (which was wrapping my `table`) and great to know about the content option. I didn't quite catch that first time around!

**Comment by jamesrswift** [comment]
Don't forget to use square braces rather than quotes if you want to do fancy content stuff

**Reply by Andrew-S-Rosen** [reply]
Thanks for the reminder! I admittedly forgot about this!
